core/FoundDefinitions.h

```diff
@@ -184,6 +189,20 @@ struct FoundModifier {
 };
 CheckSize(FoundModifier, 24, 4);
 
+struct FoundField {
+    enum class Kind : uint8_t {
+        Class,
+        Instance,
```

Comment:
Minor, but on first read I didn't realize this was related to `UnresolvedIdent::Kind::{Class,Instance}`, I thought it was like the `isSelfMethod` flag for methods.

I wonder if we should call this `ClassVariable` / `InstanceVariable`?